llvm/PKGBUILD

```diff
@@ -102,25 +102,54 @@ build() {
     -DSPHINX_WARNINGS_AS_ERRORS=OFF
   )
 
+  export CC=clang
+  export CXX=clang++
```

Comment:
if we want to use clang as compiler and thin lto, then we would also want to use whole llvm stack